### PR TITLE
staff user guide

### PR DESCRIPTION
**Pull request by DaliNeji**

### CODE REVIEW

File: docs/staff-guide/index.md [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
-melioribus vultus tecta cruorem vagina non de servo Philemon!
+Welcome to Randev. In this guide we are about to show you what each section is for as a staff member.
+
+Follow the steps below and you will be able to enjoy using Randev in no time.
 
 ## Store owner invitation
 
+In order to become a staff member in a store you need to get a store owner invitation first which will show up in your notification section that you can accept or decline.
+
+To do so you need to open the Randev application
```
Comment (safa-rekik) [inline docs/staff-guide/index.md:11]: Add the bullet list

File: docs/staff-guide/index.md [hunk 2]
```diff
@@ -1,58 +1,59 @@
...
-mundi referitur et veri credit, arbore cognoscere plausus adest.
+If a client showed up without a booking or simply that is not a Randev user you can simply instantly book for him.
+
+To do so open the Randev application, 
+
+click on the drop down menu icon, 
+
+click on my store then instant booking.
+
+it will redirect you to a page where you should provide the phone number of the client, an optional name field, which service does he want and there is a de-activated button that when clicked brings up the calendar in case he wants the booking for another day
```
Comment (safa-rekik) [inline docs/staff-guide/index.md:29]: Elaborate this in the same way I did in the french section, and do not forget using the bullet lists of markdown

File: docs/staff-guide/index.md [hunk 3]
```diff
@@ -1,58 +1,59 @@
...
-petit, lacrimisque quodque placet, invito. Concipiunt copia.
+click on my work schedule.
 
+<p align="center"><img src=./img/calendar.png><p>
 
 ## Sharing store with clients
 
-Dextera iuvenes. O sublimia nec Morphea! Reddidit robore fores attigit. Omni et
-tecta! Sive nunc petit presserat *iamque*: novorum foret.
+You can always share your store with your clients via link or via QR code so what are you waiting for?
```
Comment (safa-rekik) [inline docs/staff-guide/index.md:55]: As usual, waiting for some user guide to explain this xD

File: docs/staff-guide/index.fr.md [hunk 4]
```diff
@@ -1,58 +1,37 @@
...
+Vous devez verifier que votre calendrier soit à jour et qu'il dispose de politiques claires pour éviter tout problème .
 
+### Consulter mon calendrier
 
-Et nequeat Belides ranae, bella amore? Quem fallere facta, crimine carent,
-[illic](http://www.conata-pereo.org/sanguinisrursus), has eosdem iterum
-sinuatoque. Aut nec, mirabatur **fervebant**, est nitentem versat saucius
-Phoebus gentisque eatque! Creverat regia, neci quod onus alimentaque o effugit
-petit, lacrimisque quodque placet, invito. Concipiunt copia.
+Vous pouvez acceder a votre calendrier et voir toutes vos reservations .
```
Comment (safa-rekik) [inline docs/staff-guide/index.fr.md:27]: plus de details STP ! il faut surtout souligner qu'il faut cliquer sur le boooking gris pour ouvrir les details du booking (fait de même pour l'anglaais)

**Review by safa-rekik — CHANGES_REQUESTED** [review]
please check ly comments there.

**Review by safa-rekik — CHANGES_REQUESTED** [review]
@DaliNeji A part from the detailed comments dow there, these are the overall remarks:

- Arabic section, You need to investigate a solution for the ugly display of the bullet lists. And concerning the content, check it again with Mohammad Regaieg.
- For the french and english section : 
  Make sure to install a grammar/spelling check in your editor, pay attention to details such as capital letters and slight mistake (Most of them are made by me I admit 😅 )

File: docs/staff-guide/index.fr.md [hunk 5]
```diff
@@ -38,13 +39,19 @@ vous pouvez consulter vos réservations à tout moment et vérifier quand vous 
...
-- Cliquer sur mon emploi du temps
+- Ouvrez l'application Randev
+- Cliquez sur l'icone du menu déroulant
+- Cliquez sur programme
+- Cliquez sur mon emploi du temps
 
-<p align="center"><img src=./img/calendar.fr.png><p>
+<p align="center"><img src=./img/advancedcalender.fr.png><p>
+
+Vous pouvez changer l'affichage des rendez-vous en cliquant sur le boutton a droite en haut de la page.
```
Comment (safa-rekik) [inline docs/staff-guide/index.fr.md:53]: bouton xD

File: docs/staff-guide/index.fr.md [hunk 6]
```diff
@@ -38,13 +39,19 @@ vous pouvez consulter vos réservations à tout moment et vérifier quand vous 
...
-Vous pouvez acceder a votre calendrier et voir toutes vos reservations .
+Vous pouvez accéder a votre calendrier et voir toutes vos réservations .
 
 Pour faire ceci vous devez
 
-- Ouvrir l'application Randev
-- Cliquer sur l'icone du menu déroulant
-- Cliquer sur programme
-- Cliquer sur mon emploi du temps
+- Ouvrez l'application Randev
```
Comment (safa-rekik) [inline docs/staff-guide/index.fr.md:46]: Ay ay ay, c'est l'infinitif qu'il faut utiliser après vous devez !!!!

**Review by safa-rekik — COMMENTED** [review]
@DaliNeji Almost done, just correct some spelling (I highly recommend you configure your ide so that it does this for you!)
and please, change the instant booking screen shots so that the book button is shown (I should have made this remark in the last review, sorry)

File: docs/staff-guide/index.fr.md (hunk 6 above)
Comment (DaliNeji) [reply docs/staff-guide/index.fr.md:46]: > Ay ay ay, c'est l'infinitif qu'il faut utiliser après vous devez !!!!

It was like that but then I changed it I don't know why I did that

**Review by safa-rekik — APPROVED** [review]
This user guide is acceptable

**Pull request merged**